src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -105,7 +107,7 @@ public static uint RoundUpToPowerOf2(uint value)
 #endif
             }
 
-            // Based on https://graphics.stanford.edu/~seander/bithacks.html#RoundUpPowerOf2
+            // Based on
```

Comment:
Why did you remove the link?

Reply:
Hi @stephentoub,

thanks for bringing this to my attention. Have removed this by mistake. I have put the comment back into `BitOperations.cs`.

Best regards
Robin